packages/template-cli/templates/src/ssr/.npmrc

```diff
@@ -1,2 +1,13 @@
+; begin auth token
+//pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/registry/:username=amido-dev
+//pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/registry/:_password=djJkbGJwYXJleHViNXJuNXZ5czZva2Z6MnhoYXh2ZXBuanp6d2tjaW5ieTZxNmRteHh4cQ==
+//pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/registry/:email=npm requires email to be set but doesn't use the value
+//pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/:username=amido-dev
+//pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/:_password=djJkbGJwYXJleHViNXJuNXZ5czZva2Z6MnhoYXh2ZXBuanp6d2tjaW5ieTZxNmRteHh4cQ==
```

Comment:
Can you please remove?

Reply:
damn I knew there was something I forgot
👍